style.css

```diff
@@ -255,6 +259,8 @@ header {
   border: 0;
   height: 48px;
   background: #ff6b00;
+  transition: background-color 0.5s ease;
+  display: none;
```

Comment:
- [ ] As previously pointed out by the last reviewer,  Kindly write DRY CSS.  I counted 5 instances of the `display: none` rule. Please create a hidden class with the display none property and assign it to all elements you do not want to display 😓 
```
.hidden  {
  display: none;
}
```

- [ ] Please  Kindly apply this comcept to all other instances of code duplication in your CSS file 🤝